### PR TITLE
[Release] azdev 0.1.9

### PR DESCRIPTION
**Pull request by haroldrandom**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+0.1.9
+++++++
+* azdev extension publish: add `storage-account-ley`
```
Comment (Juliehzl) [inline HISTORY.rst]: typo for ”ley“？
Actually it should be “add --storage-acocunt-key and remove --storage-subscription".

Comment (haroldrandom) [reply HISTORY.rst]: Fixed

File: HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.1.9
+++++++
+* azdev extension publish: add --storage-acocunt-key and remove --storage-subscription
```
Comment (Juliehzl) [inline HISTORY.rst]: typo for "account"

Comment (haroldrandom) [reply HISTORY.rst]: Fixed

File: HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.1.9
+++++++
+* azdev extension publish: add --storage-account-key and remove --storage-subscription
+* azdev extension update-index: fix waring
```
Comment (Juliehzl) [inline HISTORY.rst]: warning?

Comment (haroldrandom) [reply HISTORY.rst]: https://github.com/Azure/azure-cli-dev-tools/pull/156

**Pull request merged**